authentication/mixins.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.serializers import ModelSerializer
+from rest_framework import serializers
+from rest_framework_jwt.settings import api_settings
+from rest_framework.permissions import IsAuthenticated
+
+
+class JWTTokenMixin(ModelSerializer):
+
```

Comment:
it's just a helper Mixin, don't need to inherit Model Serializer.

Reply:
Done